io.github.arunsivaramanneo.GPUViewer.yml

```diff
@@ -1,7 +1,7 @@
 id: io.github.arunsivaramanneo.GPUViewer
 runtime: org.freedesktop.Platform
 sdk: org.freedesktop.Sdk
-runtime-version: "18.08"
+runtime-version: "20.08"
 finish-args:
   - --device=all # OpenCL requires /dev/nvidia-uvm
```

Comment:
I guess this can be replaced with `--device=dri` since flatpak/flatpak@989501412419f37328b5e732b1ee8947ac801998

Reply:
Done.